### PR TITLE
My third dataset

### PR DESCRIPTION
**Pull request by CalculusC**
A labelled RAG dataset based off an article: The History Began from AlexNet: A Comprehensive Survey on Deep Learning Approaches.

### CODE REVIEW

File: llama_hub/llama_datasets/history_of_alexnet/card.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name": "History of Alexnet Dataset", "description": "A labelled RAG dataset based off an article, The History Began from AlexNet: A Comprehensive Survey on Deep Learning Approaches, by Md Zahangir Alom, Tarek M. Taha, Christopher Yakopcic, Stefan Westberg, Paheding Sidike, Mst Shamima Nasrin, Brian C Van Esesn, Abdul A S. Awwal, Vijayan K. Asari, consisting of queries, reference answers, and reference contexts.", "numberObservations": 160, "containsExamplesByHumans": false, "containsExamplesByAi": true, "sourceUrls": [], "baselines": [{"name": "llamaindex", "config": {"chunkSize": 1024, "llm": "gpt-3.5-turbo", "similarityTopK": 2, "embedModel": "text-embedding-ada-002"}, "metrics": {"contextSimilarity": 0.931, "correctness": 4.434, "faithfulness": 0.963, "relevancy": 0.931}, "codeUrl": ""}]}
```
Comment (nerdai) [inline llama_hub/llama_datasets/history_of_alexnet/card.json:1]: kindly format this json string

Comment (nerdai) [reply llama_hub/llama_datasets/history_of_alexnet/card.json:1]: also, you have a baseline script it appears. Please add it to the field `codeUrl`

Comment (nerdai) [reply llama_hub/llama_datasets/history_of_alexnet/card.json:1]: also, can you fill in the value for "sourceUrls"?

**Review by nerdai — APPROVED** [review]
minor changes. Thanks for the new dataset!

File: llama_hub/llama_datasets/history_of_alexnet/card.json (hunk 1 above)
Comment (CalculusC) [reply llama_hub/llama_datasets/history_of_alexnet/card.json]: Thanks. I have include the source urls into the card.json file.

**Review by nerdai — APPROVED** [review]
LGTM!

**Pull request merged**